corehq/apps/app_manager/tests/test_suite_multi_select_case_list.py

```diff
@@ -398,12 +392,34 @@ def test_select_parent_that_selects_other_different_case_type(self):
         self.set_parent_select(self.m1, self.m0)
 
         self.assert_module_datums(self.m0.id, [
-            ('datum', 'parent_selected_cases'),
+            ('datum', 'parent_id'),
             ('instance-datum', 'selected_cases')
         ])
         self.assert_module_datums(self.m1.id, [
-            ('datum', 'parent_selected_cases'),
+            ('datum', 'parent_id'),
             ('instance-datum', 'selected_cases'),
             ('datum', 'case_id'),
         ])
         self.assert_form_datums(self.m1f0, 'case_id')
+
+    def test_child_module_selects_other_parent_same_type(self):
+        self.set_parent_select(self.m1, self.m2)
+
+        self.assert_module_datums(self.m1.id, [
+            ('datum', 'case_id_beneficiary'),
+            ('datum', 'case_id')
+        ])
+        # This is wrong - get_add_case_preloads_case_id_xpath expects the child
+        # module datum to be renamed because of a conflict with the parent, but
+        # in this case the parent (m0) doesn't conflict, as it's a multiselect.
+        self.assert_form_datums(self.m1f0, 'case_id_beneficiary')
```

Comment:
The comment describes the issue here.  It's especially confusing, as the parent module's datum doesn't get pulled in to the suite.